Maple2.Database/Model/CharacterConfig.cs

```diff
@@ -18,6 +18,7 @@ internal class CharacterConfig {
     public IList<Wardrobe>? Wardrobes { get; set; }
     public IDictionary<StatAttribute, int>? StatAllocation { get; set; }
     public SkillBook? SkillBook { get; set; }
+    public ISet<int> FavoriteStickers { get; set; }
```

Comment:
the way you make things nullable in the schema is by setting it here:
```cs
public ISet<int>? FavoriteStickers { get; set; }
```

You can regenerate the migration afterwards to make sure it's consistent.